src/test/java/org/kohsuke/github/GitHubWireMockRule.java

```diff
@@ -241,6 +241,16 @@ protected void before() {
                     .stubFor(any(anyUrl()).willReturn(aResponse().withTransformers(ProxyToOriginalHostTransformer.NAME))
                             .atPriority(100));
         }
+
+        this.apiServer()
+                .stubFor(post(urlMatching("/repos/.*/forks")).willReturn(aResponse().withStatus(202)
+                        .withHeader("Content-Type", "application/json")
+                        .withBodyFile("fork.json")));
+
+        this.apiServer()
+                .stubFor(get(urlMatching("/repos/hub4j-test-org/github-api")).willReturn(aResponse().withStatus(200)
+                        .withHeader("Content-Type", "application/json")
+                        .withBodyFile("repository.json")));
```

Comment:
See https://github.com/hub4j/github-api/blob/main/CONTRIBUTING.md#using-wiremock-and-snapshots .  You don't need special cases hard-wired into this class.  I'm pretty sure you can record snapshots just fine. 

```suggestion
```